packages/node-app/src/routes/giveaway.route.ts

```diff
@@ -24,11 +25,11 @@ const upload = multer({ storage });
 
 export const router = express.Router();
 
-router.get('/', listGiveaways);
-router.post('/', upload.single('image'), createGiveaway);
-router.get('/:id', getGiveaway);
-router.put('/:id', upload.single('image'), updateGiveaway);
-router.put('/:id/participants', addParticipant);
-router.put('/:id/participants/:participantId', updateParticipant);
-router.get('/:id/participants', getParticipants);
-router.get('/:id/generate-winners', generateWinners);
+router.get('/', verifyToken, listGiveaways);
```

Comment:
I'm not 100% sure but I think we can just add `router.all('*', verifyToken);` instead of having to add it manually since all routes in this router will need to verify the token